openmc/tallies.py

```diff
@@ -470,14 +559,316 @@ def std_dev(self):
+        if not self._sp_filename:
+            return None
+        if self._higher_moments:
+            n = self.num_realizations
+            sum1 = self.sum
+            sum2 = self.sum_sq
+            sum3 = self.sum_third
+            sum4 = self.sum_fourth
+            self._vov = np.zeros_like(sum1, dtype=float)
+
+            # Calculate the variance of the variance (Eq. 2.232 in
+            # https://doi.org/10.2172/2372634)
+            numerator = (sum4 - (4.0*sum3*sum1)/n
+                        + (6.0*sum2*(sum1**2))/(n**2)
+                        - (3.0*(sum1)**4)/(n**3))
+            denominator = (sum2 - (1.0/n)*(sum1**2))**2
+
+            mask = denominator > 0.0
+
+            self._vov[mask] = numerator[mask]/denominator[mask] - 1.0/n
+
+            if self.sparse:
+                self._vov = sps.lil_matrix(self._vov.flatten(),
+                                           self._vov.shape)
+
+        if self.sparse:
+            return np.reshape(self._vov.toarray(), self.shape)
+        else:
+            return self._vov
+
+    @property
+    def m2(self):
+        n = self.num_realizations
+        return self.sum_sq/n - self.mean**2
+
+    @property
+    def m3(self):
+        if not self._higher_moments:
+            raise ValueError("Third central moment is not available unless "
+                             "higher_moments is True.")
+        n = self.num_realizations
+        mean = self.mean
+        sum2 = self.sum_sq/n
+        sum3 = self.sum_third/n
+
+        return sum3 - 3.0*mean*sum2 + 2.0*mean**3
+
+    @property
+    def m4(self):
+        if not self._higher_moments:
+            raise ValueError("Fourth central moment is not available unless "
+                             "higher_moments is True.")
+        n = self.num_realizations
+        mean = self.mean
+        sum2 = self.sum_sq/n
+        sum3 = self.sum_third/n
+        sum4 = self.sum_fourth/n
+
+        return sum4 - 4.0*mean*sum3 + 6.0*(mean**2)*sum2 - 3.0*mean**4
+
+    def skew(self, bias=False) -> np.ndarray:
+        """Return the sample skewness of each tally bin.
+
+        This method computes and returns the unadjusted or adjusted
+        Fisher-Pearson coefficient of skewness.
+
+        Parameters
+        ----------
+        bias : bool
+            If False, calculations are corrected for bias and the adjusted
+            Fisher-Pearson skewness (:math:`G_1`) is returned. If True,
+            calculations are not corrected for bias and the unadjusted skewness
+            (:math:`g_1`) is returned.
+
+        Returns
+        -------
+        float
+            The skewness of each tally bin
+        """
+        n = self.num_realizations
+        m2 = self.m2
+        m3 = self.m3
+
+        with np.errstate(divide="ignore", invalid="ignore"):
+            g1 = np.where(m2 > 0.0, m3/(m2**1.5), 0.0)
+
+        if bias:
+            return g1
+        else:
+            return np.where(n > 2, sqrt(n*(n - 1))/(n - 2)*g1, 0.0)
```

Comment:
This doesn't really make sense to me: the first argument to `np.where` should be array-like, not a scalar boolean